src/tests/test_filters.cpp

```diff
@@ -480,11 +482,11 @@ class Filter_Tests : public Test
 
       Test::Result test_threaded_fork()
          {
-      	Test::Result result("Threaded_Fork");
+         Test::Result result("Threaded_Fork");
 
 #if defined(BOTAN_TARGET_OS_HAS_THREADS) && defined(BOTAN_HAS_CODEC_FILTERS) && defined(BOTAN_HAS_SHA2_32)
          Botan::Pipe pipe(new Botan::Threaded_Fork(new Botan::Hex_Encoder,
-                                                   new Botan::Base64_Encoder));
+                          new Botan::Base64_Encoder));
```

Comment:
New indentation seems wrong, easy to misread as the Pipe taking two filters (`Threaded_Fork(Hex_Encoder)` and `Base64_Encoder`). Maybe best to put both `new Hex_Encoder` and `new Base64_Encoder` on their own lines.